source/application/AMPIC887DriverServer/AMGCS2.h

```diff
@@ -0,0 +1,107 @@
+#ifndef AMGCS2_H
+#define AMGCS2_H
+
+#include <QChar>
+/*!
+  * Namespace which contains all the enumerators for use with the GCS2.0 syntax
+  * provided by the PI C887.11 controllers.
+  */
+namespace AMGCS2 {
+
+	/***************
+	  * Enums
+	  ****************************/
+
+	/*!
+	  * Enumerates all the hexapod axes controlled by a C887.11 controller.
+	  */
+	enum Axis {
+		UnknownAxis = -1,
+		XAxis = 0,
```

Comment:
Originally this started as just X, but then once I'd added the movement status and recording source enums, all of them would have contained an AMGCS2::X. In the end I had to be this specific to stop the ambiguity for the compiler as much as the programmer.